pages/getting-started.md

```diff
@@ -31,7 +31,7 @@ To build the latest version of Cutter on your local machine, please follow the i
 You can contact the *Cutter* developers and community on:
 
 -  Telegram: https://t.me/cutter_re
--  #cutter on irc.freenode.net
+-  #cutter on https://web.libera.chat
```

Comment:
Here better mention `#cutter on irc.libera.chat` because that's the main way to connect, and then maybe between parenthesis the direct link `https://web.libera.chat`

Reply:
```suggestion
-  IRC: [#cutter](https://web.libera.chat/#cutter) on irc.libera.chat```